src/AGPL-1.0.xml

```diff
@@ -0,0 +1,273 @@
+<SPDX name="Affero General Public License v1.0" identifier="AGPL-1.0" osi-approved="false">
+  <urls>
+    <url>http://www.affero.org/oagpl.html</url>
+  </urls>
+  <license>
+    <title>
+      <p>AFFERO GENERAL PUBLIC LICENSE</p>
+    </title>
+    <copyright> Version 1, March 2002
```

Comment:
should the version number and license date be part of title or copyright tag... ?